cpp/src/rolling/rolling.cu

```diff
@@ -104,6 +110,103 @@ process_rolling_window(column_device_view input,
   return output_is_valid;
 }
 
+/**
```

Comment:
Not sure if this already had documentation, if not please add.

Reply:
Added the `@brief` for these functions.